packages/orbit-components/src/List/__tests__/index.test.tsx

```diff
@@ -21,6 +21,5 @@ describe("List", () => {
 
     expect(screen.getByTestId(dataTest)).toBeInTheDocument();
     expect(screen.getByText(content)).toBeInTheDocument();
-    expect(screen.getByRole("list")).toHaveStyle({ marginBottom: "12px" });
```

Comment:
Why remove this test? 🤔

Reply:
because it tests that the component is doing what it is doing. the assert has no value, outside of binding styling to unit (non-visual) tests